src/positions.ts

```diff
@@ -26,9 +26,13 @@ export interface PositionEvent {
 }
 
 export { DOMFunctions, DiffPart }
-export const findPositionsFromEvents = (options: DOMFunctions) => (
-    elements: Subscribable<HTMLElement>
-): Observable<PositionEvent> =>
+export const findPositionsFromEvents = ({
+    domFunctions,
+    tokenize = false,
```

Comment:
I'd default this to `true` so it's a non-breaking change (we provide it everywhere anyway)

Reply:
✅ 	Changed default to `true`